interface/forms/care_plan/new.php

```diff
@@ -23,7 +23,7 @@
 use OpenEMR\Core\Header;
 
 $returnurl = 'encounter_top.php';
-$formid = 0 + (isset($_GET['id']) ? $_GET['id'] : 0);
+$formid = 0 + (isset($_GET['id']) ? $_GET['id'] : '');
```

Comment:
hi @zbig01 ,
Just making sure you didn't do this on purpose for bug fixes. If so, then will ignore them.
(these were recently changed to 0 by @sjpadgett to fix PHP7 warnings)
thanks,
-brady

Reply:
I had updated this to reflect what was on the master on 11/16/2017
commit c6c5dc1ba7298409d53ef5eafd98f59052f7733a
Author: Brady Miller <brady.g.miller@gmail.com>
Date:   Thu Nov 16 00:36:50 2017 -0800

Reply:
Thanks for the explanation. Then will note bring these specific changes into codebase. The change to 0 in these places was recently done by Jerry to fix a PHP warning.